composer.json

```diff
@@ -18,10 +18,12 @@
     }
   ],
   "require": {
-    "php" : ">=7.3",
+    "php": ">=7.3",
     "ext-curl": "*",
     "ext-json": "*",
-    "ext-simplexml": "*"
+    "ext-simplexml": "*",
+    "illuminate/support": "^8.42",
```

Comment:
Was it really necessary to install this package just to make chain radom? We become dependent on an external package just to randomly generate a chain. In addition it weighs 283 KB. I think that for now we can do without this package because it has a weak use case.

Reply:
Its also needed for the env function. Without it the environment variables would only be accessed through a super global $_ENV.

The env function provides a more secure way to access .env data, an it also allows for default value if non is set:

```
env('APP_ENV', 'local');
```


That was the main reason for using the package.

Reply:
No I'm talking about illuminate/support

Reply:
We added illuminate/support as a dependency just for this use case, I don't think that's really necessary. This is what I'm talking about.
`Str::random(6)`

Reply:
Iluminate Support also comes with env(). env() is not part of DOTENV

Reply:
I can remove it if you want.

Reply:
I will also remove the illuminate Support since we won't be using .env any more and just take what we need from it.

Reply:
great !